### PR TITLE
Update elezar profile

### PR DESCRIPTION
**Pull request by elezar**
I have added some additional info to my Bio page. 

@frontendherodk @dami-gg @hesserp Let me know if it looks OK.

### CODE REVIEW

**Comment by frontendherodk** [comment]
@elezar I think you're missing some lunch in your task list

**Comment by hesserp** [comment]
:ship:

**Comment by elezar** [comment]
@frontendherodk How about now?

File: profiles/elezar.md [hunk 1]
```diff
@@ -9,5 +9,8 @@ The quick brown fox jumps over the lazy dog.
 ## TODOs for today:
 - [x] Have coffee
 - [ ] Attend GitHub for Developers training
+- [x] Lunch
+    - [x] Dolores
```
Comment (frontendherodk) [inline profiles/elezar.md:13]: Nice one!

**Comment by frontendherodk** [comment]
👍 🚢

**Pull request merged**